.github/workflows/test.yml

```diff
@@ -348,6 +348,7 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: 3.0
```

Comment:
Any idea why we're using Ruby 3 in the `docs_test` job but Ruby 2.4 in the `ruby_test` job?

Reply:
No idea. Seems not good. Will try downgrading here and see what happens. If it doesn't work, I may punt on it though

Reply:
Seems to work